### PR TITLE
Add changelog for 6.4.1

### PR DESCRIPTION
**Pull request by oskosk**
This adds a changelog entry for #10001.

#### Changes proposed in this Pull Request:

* Updates readme.txt to include changelog for 6.4.1
* Bumps stable tag to `6.4`

#### Testing instructions

none needed

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,5 +1,38 @@
 == Changelog ==
 
+= 6.4 =
```
Comment (dereksmart) [inline changelog.txt]: Since it's only been a day, let's keep this in readme.txt as well

File: readme.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 Contributors: automattic, adamkheckler, aduth, akirk, allendav, alternatekev, andy, annezazu, apeatling, azaozz, batmoo, barry, beaulebens, blobaugh, cainm, cena, cfinke, chaselivingston, chellycat, clickysteve, csonnek, danielbachhuber, davoraltman, daniloercoli, designsimply, dllh, drawmyface, dsmart, dzver, ebinnion, eliorivero, enej, eoigal, erania-pinnera, ethitter, gcorne, georgestephanis, gibrown, goldsounds, hew, hugobaeta, hypertextranch, iammattthomas, iandunn, jblz, jasmussen, jeffgolenski, jeherve, jenhooks, jenia, jessefriedman, jgs, jkudish, jmdodd, joanrho, johnjamesjacoby, jshreve, keoshi, koke, kraftbj, lancewillett, lschuyler, macmanx, martinremy, matt, matveb, mattwiebe, maverick3x6, mcsf, mdawaffe, MichaelArestad, migueluy, mikeyarce, mkaz, nancythanki, nickmomrik, obenland, oskosk, pento, professor44, rachelsquirrel, rdcoll, ryancowles, richardmuscat, richardmtl, roccotripaldi, samhotchkiss, scarstocea, sdquirk, stephdau, tmoorewp, tyxla, Viper007Bond, westi, yoavf, zinigor
 Tags: Jetpack, WordPress.com, backup, security, related posts, CDN, speed, anti-spam, social sharing, SEO, video, stats
-Stable tag: 6.3.3
+Stable tag: 6.4
```
Comment (dereksmart) [inline readme.txt]: is the tag already created in svn?

Comment (oskosk) [reply readme.txt]: yeah . yesterday

Comment (dereksmart) [reply readme.txt]: oh right I parsed this as 6.4.1 for no reason

**Comment by oskosk** [comment]
Ported to `branch-6.4` in acd977c0ab0428425fe119173baedf72aa05663a

**Pull request merged**